modern/src/reports/EventReportPage.js

```diff
@@ -59,30 +59,38 @@ const EventReportPage = () => {
 
   const [items, setItems] = useState([]);
 
+  const columns = [{
+    headerName: t('positionFixTime'),
+    field: 'serverTime',
+    type: 'dateTime',
+    flex: 1,
+    valueFormatter: ({ value }) => formatDate(value),
+  }, {
+    headerName: t('sharedType'),
+    field: 'type',
+    type: 'string',
+    flex:1,
+  }, {
+    headerName: t('sharedGeofence'),
+    field: 'geofenceId',
+    type: 'number',
+    flex: 1,
+  }, {
+    headerName: t('sharedMaintenance'),
+    field: 'maintenanceId',
+    type: 'number',
+    flex: 1
+  }];
+
   return (
     <ReportLayoutPage filter={<Filter setItems={setItems} />}>
-      <TableContainer component={Paper}>
-        <Table>
-          <TableHead>
-            <TableRow>
-              <TableCell>{t('positionFixTime')}</TableCell>
-              <TableCell>{t('sharedType')}</TableCell>
-              <TableCell>{t('sharedGeofence')}</TableCell>
-              <TableCell>{t('sharedMaintenance')}</TableCell>
-            </TableRow>
-          </TableHead>
-          <TableBody>
-            {items.map(item => (
-              <TableRow key={item.id}>
-                <TableCell>{formatPosition(item, 'eventTime')}</TableCell>
-                <TableCell>{item.type}</TableCell>
-                <TableCell>{}</TableCell>
-                <TableCell>{}</TableCell>
-              </TableRow>
-            ))}
-          </TableBody>
-        </Table>
-      </TableContainer>
+      <Paper>
```

Comment:
Can we move paper part to the `ReportLayoutPage` instead of duplicating it?

Reply:
Yes. Done